js_modules/dagster-ui/packages/ui-core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -34,21 +35,16 @@ import {buildRepoAddress} from '../workspace/buildRepoAddress';
 import {RepoAddress} from '../workspace/types';
 import {workspacePathFromAddress} from '../workspace/workspacePath';
 
-import {
-  LiveDataForNode,
-  displayNameForAssetKey,
-  GraphNode,
-  nodeDependsOnSelf,
-  stepKeyForAsset,
-} from './Utils';
+import {displayNameForAssetKey, GraphNode, nodeDependsOnSelf, stepKeyForAsset} from './Utils';
 import {SidebarAssetQuery, SidebarAssetQueryVariables} from './types/SidebarAssetInfo.types';
 import {AssetNodeForGraphQueryFragment} from './types/useAssetGraphData.types';
 
 export const SidebarAssetInfo: React.FC<{
   graphNode: GraphNode;
-  liveData?: LiveDataForNode;
-}> = ({graphNode, liveData}) => {
+}> = ({graphNode}) => {
   const {assetKey, definition} = graphNode;
+  const {liveDataByNode} = useAssetsLiveData(React.useMemo(() => [assetKey], [assetKey]));
+  const liveData = liveDataByNode[JSON.stringify(assetKey.path)];
```

Comment:
Would be nice to use one of our existing tokenization strategies like `tokenForAssetKey`